gh-pages/index.md

```diff
@@ -51,106 +27,34 @@ Options:
       User Token for connecting to remote sonar instance
 ```
 
-CodeCharta may ask for a user authentification, depending on the configuration of the projects SonarQube server. Log in to your server, navigate to the project and create a user token. This string can be easily added as seen in the command description above.  
+CodeCharta may ask for a user authentification, depending on the configuration of the projects SonarQube server. 
+Log in to your server, navigate to the project and create a user token. This string can be easily added as seen in the command description above.
 If you don't have any own projects yet, you can test CodeCharta with any opensource project you can find on the sonarqube website.
 
 The easiest example:
-  - >./ccsh sonarimport [sonar-server-url] [project-id]
+> ./ccsh sonarimport [sonar-server-url] [project-id]
  
 An output file has been created.
 
-## Visualize the analyzed project
+## Visualizing the analyzed project
 
-Open the visualization directory and run the executable
-A new window should open now. 
-![Image](images/screenshot_visu.PNG)
+Open the visualization directory and run the executable. A new window should open now. 
+You can also use the [online demo](https://maibornwolff.github.io/codecharta/visualization/app/index.html?file=codecharta.cc.json).
```

Comment:
@mrpatpat but actually there still is the online demo (although its another file)